src/test/java/fi/aalto/cs/apluscourses/ui/ideactivities/TaskViewTest.java

```diff
@@ -0,0 +1,39 @@
+package fi.aalto.cs.apluscourses.ui.ideactivities;
+
+import fi.aalto.cs.apluscourses.model.task.OpenFileTask;
+import fi.aalto.cs.apluscourses.presentation.ideactivities.TaskViewModel;
+import org.jetbrains.annotations.NotNull;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class TaskViewTest {
```

Comment:
This test class is a bit confusing because
1. we shouldn't test views as they should be free of logic
2. the method that is tested (`getTaskDescriptionText`) is overridden by the mock object (`TestTaskView`). What is actually tested here.